--- colord-1.4.1/data/meson.build.orig	2017-08-21 11:02:36.000000000 +0200
+++ colord-1.4.1/data/meson.build	2017-12-23 22:57:10.663716927 +0100
@@ -14,7 +14,7 @@
 
 if get_option('enable-bash-completion')
   install_data('colormgr',
-    install_dir: bash_completion.get_pkgconfig_variable('completionsdir')
+    install_dir: get_option('with-bash-completion-dir')
   )
 endif
 
--- colord-1.4.1/meson.build.orig	2017-08-21 11:02:36.000000000 +0200
+++ colord-1.4.1/meson.build	2017-12-23 23:41:23.647019972 +0100
@@ -146,9 +146,6 @@
   gnome_desktop = dependency('gnome-desktop-3.0', version : '>= 0.2.7')
   colord_gtk = dependency('colord-gtk', version : '>= 0.1.24')
 endif
-if get_option('enable-bash-completion')
-  bash_completion = dependency('bash-completion', version : '>= 2.0')
-endif
 
 if get_option('enable-sane')
   sane = dependency('sane-backends')
--- colord-1.4.1/meson_options.txt.orig	2017-08-21 11:02:36.000000000 +0200
+++ colord-1.4.1/meson_options.txt	2017-12-23 23:42:47.510352342 +0100
@@ -12,3 +12,5 @@
 option('with-daemon-user', type : 'string', value : 'root', description : 'User for running the colord daemon')
 option('enable-man', type : 'boolean', value : true, description : 'Generate man pages')
 option('enable-docs', type : 'boolean', value : true, description : 'Generate documentation')
+option('with-bash-completion-dir', type : 'string', description : 'Bash
+completions dir')
