--- colord-1.4.1/data/meson.build.orig	2017-08-21 11:02:36.000000000 +0200
+++ colord-1.4.1/data/meson.build	2017-12-23 22:57:10.663716927 +0100
@@ -14,7 +14,7 @@
 
 if get_option('enable-bash-completion')
   install_data('colormgr',
-    install_dir: bash_completion.get_pkgconfig_variable('completionsdir')
+    install_dir: get_option('bash_completion_dir')
   )
 endif
 
--- colord-1.4.3/meson.build.orig	2018-09-29 19:42:17.407154714 +0200
+++ colord-1.4.3/meson.build	2018-09-29 19:43:56.833820242 +0200
@@ -149,9 +149,6 @@
   gnome_desktop = dependency('gnome-desktop-3.0', version : '>= 0.2.7')
   colord_gtk = dependency('colord-gtk', version : '>= 0.1.24')
 endif
-if get_option('bash_completion')
-  bash_completion = dependency('bash-completion', version : '>= 2.0')
-endif
 
 if get_option('sane')
   sane = dependency('sane-backends')
--- colord-1.4.3/meson_options.txt.orig	2018-09-29 19:42:17.410488047 +0200
+++ colord-1.4.3/meson_options.txt	2018-09-29 19:43:20.723820651 +0200
@@ -1,5 +1,6 @@
 option('session_example', type : 'boolean', value : false, description : 'Enable session example')
 option('bash_completion', type : 'boolean', value : true, description : 'Enable bash completion')
+option('bash_completion_dir', type : 'string', description : 'Bash completion dir')
 option('udev_rules', type: 'boolean', value: true, description: 'Install udev rules')
 option('systemd', type : 'boolean', value : true, description : 'Enable systemd integration')
 option('libcolordcompat', type : 'boolean', value : false, description : 'Enable libcolordcompat.so which is used by ArgyllCMS')
